backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala

```diff
@@ -267,7 +268,7 @@ trait StandardAsyncExecutionActor extends AsyncBackendJobExecutionActor with Sta
         |cd $cwd
         |ENVIRONMENT_VARIABLES
         |INSTANTIATED_COMMAND
-        |) > >(tee $stdoutPath) 2> >(tee $stderrPath >&2)
+        |) $stdinRedirection > >(tee $stdoutPath) 2> >(tee $stderrPath >&2)
```

Comment:
ToL: The `> >` surprised me but since I realised it's not part of your change... 🤷‍♂️

Reply:
Yes I was surprised I didn't catch more grief for that when I actually made the change. 🙂  That syntax enables those tees to both stdout/stderr and name stdout/stderr files.